addrecipe.html

```diff
@@ -17,24 +17,25 @@
 </head>
 
 <body>
-    <nav class="uk-navbar-container" uk-navbar>
-        <div class="uk-navbar-left">
-            <ul class="uk-navbar-nav">
-                <li><a href="index.html">Home</a></li>
-                <li class="uk-active"><a href="addrecipe.html">Add recipe</a></li>
-            </ul>
-        </div>
-    </nav>
+  <nav class="uk-navbar-container" uk-navbar>
+    <div class="uk-navbar-left">
+      <ul class="uk-navbar-nav">
+        <li><a href="index.html">Home</a></li>
+        <li class="uk-active"><a href="addrecipe.html">Add recipe</a></li>
+      </ul>
+    </div>
+  </nav>
+
 
+  <form class="uk-search uk-search-default margin-medium">
     <div class="uk-margin">
-        <form class="uk-search uk-search-default">
-            <a href="" class="uk-search-icon-flip" uk-search-icon></a>
-            <input class="uk-search-input" type="search" placeholder="Search...">
-        </form>
+      <label class="font-size-medium">WHAT ARE YOU LOOKING FOR?</label>
+      <a href="#" class="uk-search-icon-flip" uk-search-icon></a>
```

Comment:
I'm a little surprised they use an empty link here, but it seems this is what they do on the framework's site ¯/\(°_o)/¯

Reply:
https://github.com/EwaTrzemzalska/zjemto/issues/34